azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -96,8 +96,9 @@
   private static final String CONF_DIR = "conf";
   private static final String JOB_THREAD_COUNT = "flow.num.job.threads";
   private static final String DEFAULT_LOG_CHUNK_SIZE = "5MB";
+  private static final String FLOW_EXECUTION_ID = "FLOW_EXECUTION_ID";
```

Comment:
These variables are already created in constants. Please reuse those.

Reply:
thanks!, did not know.